src/routes/DisciplineSelectionScreen.tsx

```diff
@@ -1,20 +1,22 @@
 import { RouteProp } from '@react-navigation/native'
 import { StackNavigationProp } from '@react-navigation/stack'
 import React from 'react'
-import { FlatList, StatusBar } from 'react-native'
+import { FlatList } from 'react-native'
 import styled from 'styled-components/native'
 
 import DisciplineListItem from '../components/DisciplineListItem'
+import RouteWrapper from '../components/RouteWrapper'
 import ServerResponseHandler from '../components/ServerResponseHandler'
 import Title from '../components/Title'
 import { Discipline } from '../constants/endpoints'
 import { useLoadDisciplines } from '../hooks/useLoadDisciplines'
 import { RoutesParams } from '../navigation/NavigationTypes'
 import { childrenDescription } from '../services/helpers'
 
-const Root = styled.View`
+const List = styled.FlatList`
   margin: 0 ${props => props.theme.spacings.md};
-`
+  height: 100%;
+` as unknown as typeof FlatList
```

Comment:
:/

Reply:
show me a suggestion instead of telling me that it's not nice. i know that by myself :/

Reply:
Sorry, was not meant that way. If I'd know another way to do this I'd have suggested it :) So all good!